flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -262,7 +265,7 @@ public void testKeyedListStateSerializerReconfiguration() throws Exception {
                         new TestType.ReconfigurationRequiringTestTypeSerializer()));
     }
 
-    @Test
+    @TestTemplate
     public void testKeyedListStateRegistrationFailsIfNewStateSerializerIsIncompatible() {
```

Comment:
```suggestion
    void testKeyedListStateRegistrationFailsIfNewStateSerializerIsIncompatible() {
```
Could be package private